CHANGES.md

```diff
@@ -23,6 +23,14 @@ OpenSSL 3.0
 
 ### Changes between 1.1.1 and 3.0 [xx XXX xxxx]
 
+ * Deprecate TS_VERIFY_CTX_add_flags and TS_RESP_CTX_add_flags have been
```

Comment:
If someone thinks this should be merged then you would need to fix up this first sentence.

Reply:
if it gets one approval i will fix the sentence else why bother.